tox.ini

```diff
@@ -133,6 +133,7 @@ alwayscopy = True
 passenv =
 	{[base]passenv}
 basepython = /home/travis/virtualenv/python3.5/bin/python3.5
+envdir = {toxworkdir}/py35
```

Comment:
Review: mitigates the impact of any confusing CI directory kluge in python3.5 caused by search path conflicts.